crates/ln-dlc-node/src/networking.rs

```diff
@@ -0,0 +1,89 @@
+use crate::node::NodeInfo;
+use futures::future;
+use lightning::ln::peer_handler::APeerManager;
+use lightning::ln::peer_handler::SocketDescriptor;
+use std::future::Future;
+use std::ops::Deref;
+use tracing::debug;
+
+#[cfg(feature = "ln_net_axum_ws")]
+pub mod axum;
+#[cfg(feature = "ln_net_tcp")]
+pub mod tcp;
+#[cfg(feature = "ln_net_ws")]
+mod tungstenite;
+
+// TODO we can let the connect functions be async as well
```

Comment:
Can you elaborate on this? I don't get what you mean.

Reply:
Oops, this is a fossil of a previous attempt. Good catch, thanks!